scripts/scrapeWalletConnectRegistry.ts

```diff
@@ -0,0 +1,26 @@
+import * as fs from 'fs'
+import fetch from 'node-fetch'
+import * as path from 'path'
+import { RegistryItem } from '../src/plugins/walletConnectToDapps/types'
+
+const alphabet = 'abcdefghijklmnopqrstuvwxyz'.split('')
+
+const registryJsonUrl =
+  'https://explorer.walletconnect.com/_next/data/CWVJGGRcXLbAilG1CUAn3/index.json'
+
+async function run() {
+  const listings: RegistryItem[] = []
+  for (const searchQuery of ['', ...alphabet]) {
+    // eslint-disable-next-line no-console
```

Comment:
Note for other reviewers: This is actually correct, we don't want to use `shapeshiftoss/logger/no-native-console` as it will hint about moduleLogger use, which we can't use here